tests/integ/test_transformer.py

```diff
@@ -351,6 +351,47 @@ def test_single_transformer_multiple_jobs(sagemaker_session, mxnet_full_version,
         )
 
 
+def test_transform_mxnet_logs(sagemaker_session, mxnet_full_version):
+    data_path = os.path.join(DATA_DIR, "mxnet_mnist")
+    script_path = os.path.join(data_path, "mnist.py")
+
+    mx = MXNet(
+        entry_point=script_path,
+        role="SageMakerRole",
+        train_instance_count=1,
+        train_instance_type="ml.c4.xlarge",
```

Comment:
a new thing since this PR was originally created - please use this fixture for the instance type: https://github.com/aws/sagemaker-python-sdk/blob/master/tests/conftest.py#L259